ci/requirements-py27-min.yml

```diff
@@ -2,8 +2,8 @@ name: test_env
 dependencies:
   - python=2.7
   - pytest
-  - numpy==1.9.3
```

Comment:
This does look a possible NumPy bug. Instead of updating the required version of numpy for everything, I would just skip it explicitly for the failing test.